webapp-django/crashstats/tokens/templates/tokens/home.html

```diff
@@ -46,7 +46,8 @@
                     <time class="ago" data-date="{{ token.created.isoformat() }}">{{ token.created.isoformat() }}</time>
                 </h4>
                 <p class="code">
-                    <code>{{ token.key[:12] }}
```

Comment:
But that's uglier than flexbox. That's a hack. 
I picked flexbox because of https://css-tricks.com/fighting-the-space-between-inline-block-elements/